index.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
         type: 'input',
         name: 'issues',
         message: dedent`
-          List any issues closed by this change:
+          List any issue(s) closed by this change (e.g. #123, #124):
```

Comment:
@BerkeleyTrue fine by me! :+1: I'd want to be a little lazy too :smile: I'll send up an update later today.